docs/index.rst

```diff
@@ -4120,6 +4119,52 @@ argument with code ``0`` to use ``last prediction`` and ``1`` for
     local_model.predict({"petal length": 3, "petal width": 1},
                         missing_strategy=PROPORTIONAL)
 
+If you would like a per-class prediction of the confidence or
+probability for each class in the model, you can use the
+``predict_probability`` and ``predict_confidence`` methods
+respectively.  Each of these methods takes the ``by_name`` and
+``missing_strategy`` arguments that function as in ``predict``, and
+one additional argument, ``compact``.
+
+If ``compact`` is ``False`` (the default), the output of these
+functions is a list of maps, each with the keys ``prediction`` and
+``probability`` mapped to the class name and its associated
+probability.
+
+So, for example, the following:
+
+.. code-block:: python
+
+    local_model.predict_probability({"petal length": 3})
+
+would result in
+
+.. code-block:: python
+
+    [{'prediction': u'Iris-setosa',
+      'probability': 0.0033003300330033},
+     {'prediction': u'Iris-versicolor',
+      'probability': 0.4983498349834984},
+     {'prediction': u'Iris-virginica',
+      'probability': 0.4983498349834984}]
+
+If ``compact`` is ``True``, only the probabilities themselves are
+returned, as a list in class name order.  So this:
```

Comment:
maybe worth mentioning here that the local_model now has a class_names attribute that contains the list of ordered class names?